src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+type IntrinsicElements = JSX.IntrinsicElements["input"];
+
+export interface RadioProps extends IntrinsicElements {
+  id: string;
+  label: string;
+  children?: JSX.Element[];
+  disabled?: boolean;
+  isTile?: boolean;
+  tileDescription?: string;
+  value?: string;
+}
+
+/**
+ * Radio Component
+ * @param {string}  id                A unique identifier for the input.
+ * @param {string}  label             Label text that appears to the right of the Radio.
+ * @param {boolean} [checked]         Set the initial checked state.
+ * @param {Array}   [children]        An array of child elements to appear when selected.
+ * @param {boolean} [disabled]        Sets the Radio to its disabled state.
+ * @param {boolean} [isTile]          Use the tile variation of the Radio.
+ * @param {string}  [tileDescription] Text that can be used to describe the label in more detail. Activates the tile variation automatically.
+ * @param {string}  [value]           Value of the input element.
+ */
+export const Radio: React.FC<RadioProps> = ({
+  children,
+  disabled = false,
+  id,
+  isTile = false,
+  label,
+  tileDescription,
+  value,
+  ...rest
+}) => {
+  return (
+    <div className="usa-radio">
+      <input
+        className={
+          isTile
+            ? "usa-radio__input usa-radio__input--tile"
+            : "usa-radio__input"
+        }
+        disabled={disabled}
+        id={id}
+        type="radio"
+        value={value}
+        {...rest}
+      />
+      <label className="usa-radio__label" htmlFor={id}>
+        {label}
+        {isTile && tileDescription && (
+          <span className="usa-radio__label-description">
+            {tileDescription}
+          </span>
+        )}
```

Comment:
I know the design only has a description for the tiled `Radio`, but do we think there's merit in just having the description universally available?
```suggestion
        {description && (
          <span className="usa-radio__label-description">
            {description}
          </span>
        )}
```

Reply:
Because that would introduce a third kind of radio button (and checkbox), and that third kind isn't part of the design system, I wouldn't want to display a description field with a non-tile checkbox unless HCD specifically asks for it.